duckduckgo/duckduckgo-autofill
# Update release workflow

**Reviewer:** @GioSensation 
**Asana:** https://app.asana.com/0/1198964220583541/1203931336731295/f

## Description
Updates to release workflow to split sections into separate jobs which can pass or fail independently.

## Steps to test
See last commit for dry-run version
Last dry-run -- https://github.com/duckduckgo/duckduckgo-autofill/actions/runs/4894548018
<img width="2040" alt="image" src="https://user-images.githubusercontent.com/635903/236495871-2073aa95-bcdb-4b6a-bbb9-f3a038bbacbf.png">
